packages/spruce-skill-server/lib/GraphQLSubscriptionServer.ts

```diff
@@ -102,6 +102,9 @@ export default class GQLSubscriptionServer {
 		const token = authorizationHeader.replace('JWT ', '')
 		const decoded = jwt.verify(
 			token,
+			// TODO: config's type doesn't describe this API correctly...
```

Comment:
@kengoldfarb Not sure what the deal is here, but noticed that the types file hasn't been updated in npm in a few years. Possibly just totally wrong. We forked this right?

Reply:
I think we should now be able to just reference it directly: `config.API_KEY` and have it strongly typed